src/hyp3_isce2/burst.py

```diff
@@ -177,7 +177,7 @@ def download_burst(asf_session: requests.Session, burst_params: BurstParams, out
     return Path(out_file)
 
 
-def spoof_safe(burst: BurstMetadata, burst_tiff_path: Path, base_path: Path = Path('.')) -> Path:
+def spoof_safe(burst: BurstMetadata, burst_tiff_path: Path, base_path: Path = Path.cwd()) -> Path:
```

Comment:
Ruff does not like `Path('.')` and would prefer you use either `Path()` or `Path.cwd()`. There were some gotcha's here around when `'.'` was resolved to an actual path, so we'll want to make sure that `Path.cwd()` doesn't hit those again....